main.py

```diff
@@ -14,7 +14,9 @@
 import os.path
 import sys, os
 
-old_coins = ["OTHERCRAP"]
+# To add a coin to ignore, add it to the json array in old_coins.json
+globals.old_coins = load_old_coins()
```

Comment:
You should add a 
```
if os.path.isfile('old_coin.json'):
   globals.old_coins = load_old_coins()
else:
   globals.old_coins = {}
```

This will intialize the global variable if no file is present.  And won't error when appending to the dictionary.